src/ripple/rpc/impl/DeliveredAmount.cpp

```diff
@@ -172,5 +172,86 @@ insertDeliveredAmount(
         std::move(serializedTx),
         transactionMeta);
 }
-} // RPC
-} // ripple
+
+template <class T>
+void
+insertDeliveredAmount(
+    rpc::v1::CurrencyAmount& proto,
+    RPC::ContextGeneric<T>& context,
+    std::shared_ptr<Transaction> transaction,
+    TxMeta const& transactionMeta)
```

Comment:
We should look at trying to get rid of this code duplication if we can. Maybe we can do it in another PR - but we should at least make a TODO to look at this (you don't need to do this now).

Reply:
Added a TODO